### ISSUE TITLE
'stack build' with ghc-8.6.1

### ISSUE THREAD

**Issue by vnluc** [haskell/haskell-ide-engine#844]
I just downloaded ghc 8.6.1 and try to run 'stack build command and found lot of error. 

Is stack build config (stack.yaml or stack-8.6.1.yaml) available yet?

**Comment by alanz** [comment]
No, there are a number of dependencies that still need updating.

**Comment by vnluc** [comment]
@alanz , thank you for updating. So we will wait for it available

**Comment by vnluc** [comment]
hi, is there any one working on building hie with ghc 8.6.1?

**Comment by Anrock** [comment]
I guess it's in #962

**Comment by bubba** [comment]
@Anrock @alanz is it safe to say we have full 8.6 support and close this?
